docker/stubs/flac/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+export CXXFLAGS="-fsanitize=address -fsanitize-coverage=edge,indirect-calls"
+export CXX="clang++"
+export CFLAGS="-fsanitize=address -fsanitize-coverage=edge,indirect-calls"
+export CC="clang"
+export LDFLAGS="-fsanitize=address -fsanitize-coverage=edge,indirect-calls"
```

Comment:
Same "ASAN + SANCOV" line here three times.

Why not:

```sh
export CC=clang
export CXX=clang++
export SANCOV="-fsanitize=address -fsanitize-coverage=edge,indirect-calls"
export CFLAGS=${SANCOV}
export CXXLAGS=${SANCOV}
export LDFLAGS=${SANCOV}
```

and consider dropping the unused exports. Why define things not in use?